src/analysis/souffle/tests/arcs_fact_tests/BUILD

```diff
@@ -39,3 +42,36 @@ ALL_DL_TEST_FILES = glob(["*.dl"])
         dl_script.replace(".dl", "_souffle_cc_library"),
     ],
 ) for dl_script in ALL_DL_TEST_FILES]
+
+[souffle_cc_library(
```

Comment:
Instead of creating a separate target, can we add something like `#define ALL_PRINCIPALS_OWN_ALL_TAGS` in the relevant test files?

Reply:
Right now, I'm trying to run this test with two variants: one in which the `derives from` claim is trusted and one in which it is untrusted. I can't do that with preprocessor macros within the test. Clearly this is not the best way to do test variants, though; I've filed #146 to come up with a better way of producing test variants in the future.